backend/app/services/execution/request_builder.py

```
@@ -0,0 +1,1074 @@
+        task: TaskResource,
+        user: User,
+        team: Kind,
+        message: str,
+        *,
+        # Feature toggles
+        enable_tools: bool = True,
+        enable_web_search: bool = False,
+        enable_clarification: bool = False,
+        enable_deep_thinking: bool = True,
+        # Skill configuration
+        skill_names: Optional[List[str]] = None,
```

Ruff (0.14.14)

[warning] Unused method argument: `skill_names`

(ARG002)